### ISSUE TITLE
It is possible to generate a metric file with empty # HELP and / or # TYPE lines

### ISSUE THREAD

**Issue by JDA88** [opsdis/aci-exporter#25]
**Describe the bug**
If you don’t input the `type` and / or `help` field in the yaml it will successfully generate the metric file but prometheus will fail to scrap it

**To Reproduce**
if you leave commented the `type` and / or `help` lines in the following metric the fille will be generated with empty `# HELP` and / or `# TYPE` lines
```
    metrics:
      - name: test_metric
        value_name: eqptCh.attributes.model
#        type: "gauge"
        unit: "ratio"
#        help: "Returns the kernel space cpu load of a fabric node"
```

**Expected behavior**
**Option1:** The program should detect the yaml file as invalid
**Option2:** The program should replace the missing filed to "default" values `type: "gauge"` / `help: "Missing"`

**Comment by thenodon** [comment]
@JDA88 - thanks for reporting this issue. If TYPE is missing Prometheus will report a scraping error as `invalid metric type ""`. So I think its a good idea to use your option 2 and document this default behavior. A missing HELP do not seem to break Prometheus, but adding a text indicating its missing is a good idea.

**Comment by thenodon** [comment]
Fix in #27
